framework/resources/Datasources/s3/minio/stop_mn.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+
+if [ ! -z "$(top -bn1 |grep minio |grep -oE '^[ ]*[0-9]+' |grep -oE '[0-9]+')" ]
+then
+  kill -9 $(top -bn1 |grep minio |grep -oE '^[ ]*[0-9]+' |grep -oE '[0-9]+')
```

Comment:
Is there no clean way to stop the server?

Reply:
Minio server is running in the background by running command "nohup ... &".
There is no way to send Minio any shutdown commands.
If we don't run Minio server in the background, the only way to stop it is to close bash window or send Ctrl+C key combination to this bash window.

Reply:
Sure, thanks for clarifying.